README.md

```diff
@@ -108,18 +108,19 @@ export default App;
+| `userNickName`          | `string`                                                                                          | No      | N/A           | The nickname of the user                                                                                                                                  |                                                                   |
+| `enableEmojiFeedback`   | `boolean`                                                                                         | No      | `true`        | Enables emoji feedback functionality.                                                                                                                     |
+| `enableMention`         | `boolean`                                                                                         | No      | `true`        | Enables mention functionality.                                                                                                                            |
+| `deviceType`      | `'desktop' \| 'mobile'`                                                                           | No      | N/A        | Device type to be used in the widget                                                                                                                      |
+| `stringSet`             | `{ [key: string]: string }`                                                                       | No      | N/A           | Customizable string set. Available string sets can be found [here](https://github.com/sendbird/sendbird-uikit-react/blob/main/src/ui/Label/stringSet.ts). |
+| `customRefreshComponent` | `{ icon: string, style: React.CSSProperties, width: string, height: string, onClick: () => void }` | No      | N/A           | Customizable refresh component. You can set properties such as `icon`, `style`, `width`, `height`, and `onClick`.                                         |
+| `configureSession`      | `() => SessionHandler`                                                                            | No      | N/A           | Function to configure the user session. The example usage can be found [here](./custom-session-guide.md). <br/><br/> Must be used with `userId`.          |
+| `autoOpen`              | `boolean`                                                                                         | No      | N/A        | Determines whether the chatbot widget automatically opens when the browser window is opened.                                                              |
+| `messageInputControls`  | `{ blockWhileBotResponding?: boolean \| number }`                                                 | No      | N/A        | Allows to control enabled/disabled state of the message input.                                                                                            |
```

Comment:
how about

```markdown
| `messageInputControls.blockWhileBotResponding` | ~~ | ~~ |
```

Reply:
For components, it might be acceptable to represent them as objects
but for options, it seems more convenient to separate each item individually.